### PR TITLE
LF: clean shifting BigNumeric builtin

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-lf/spec/daml-lf-1.rst [hunk 1]
```diff
@@ -3842,6 +3842,14 @@ BigNumeric functions
 
   [*Available in version ≥ 1.13*]
 
+* ``SHIFT_RIGHT_BIGNUMERIC : 'Int64' → 'BigNumeric' → 'Int64'``
```
Comment (cocreature) [inline daml-lf/spec/daml-lf-1.rst:3845]:
```suggestion
* ``SHIFT_RIGHT_BIGNUMERIC : 'Int64' → 'BigNumeric' → 'BigNumeric'``
```

File: daml-lf/spec/daml-lf-1.rst [hunk 2]
```diff
@@ -3842,6 +3842,14 @@ BigNumeric functions
...
+
+  Multiply the second argument by 10 to the power of the first
```
Comment (cocreature) [inline daml-lf/spec/daml-lf-1.rst:3847]:
```suggestion
  Multiply the second argument by 10 to the negative power of the first
```

**Review by cocreature — APPROVED** [review]
Thanks!

File: daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto [hunk 3]
```diff
@@ -577,10 +577,11 @@ enum BuiltinFunction {
   SCALE_BIGNUMERIC = 137; // *Available in versions >= 1.13*
   PRECISION_BIGNUMERIC = 138; // *Available in versions >= 1.13*
   ADD_BIGNUMERIC = 139; // *Available in versions >= 1.13*
+  ADD_BIGNUMERIC = 139; // *Available in versions >= 1.13*
```
Comment (sofiafaro-da) [inline daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto:580]: I think this may be duplicated by accident.

Comment (remyhaemmerle-da) [reply daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto:580]:
```suggestion
  ADD_BIGNUMERIC = 139; // *Available in versions >= 1.13*
```

Comment (remyhaemmerle-da) [reply daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto]: good catch.

**Pull request merged**